xml/http_wwwauth.xml

```diff
@@ -243,9 +378,13 @@
     <description>Ignore Negotiate-only</description>
     <example>Negotiate</example>
   </fingerprint>
-  <fingerprint pattern="^(?:Basic|Digest) .*realm=&quot;null&quot;">
-    <description>Ignore null</description>
+  <fingerprint pattern="^(?:Basic|Digest) .*realm=&quot;(?:/|\.|null|/?index.html?)?&quot;">
```

Comment:
That's the frequency of this match? Should it, for performance reasons be at the beginning as we've done in some other FP files? Would an ending anchor here be reasonable?

Reply:
It really depends on where you are looking.  I was checking across a dozen or more HTTP ports and these were not in the top 10 overall.  On some specific ports, yes, I'm sure they are higher, but thats a tricky subject out of scope for now ;)